test/features/auth/sign_in_test.rb

```diff
@@ -0,0 +1,24 @@
+require "test_helper"
+
+feature "Omniauth" do
+  scenario "sign in with github works" do
+    OmniAuth.config.test_mode = true
+    OmniAuth.config.add_mock(:github,
+                            {
+                              uid: '12345',
+                              info: {
+                                nickname: 'test_github_user',
+                                email: 'test_github_user@mail.com'
+                              },
+                              })
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.